Buildings/Experimental/DHC/EnergyTransferStations/Combined/Generation5/BaseClasses/PartialParallel.mo

```diff
@@ -387,8 +345,10 @@ as the connections are in parallel.
 </li>
 </ul>
 <p>
-Eventually, note that this hydronic layout is not compatible with a
-compressor-less cooling mode using only the district heat exchanger.
+Note that the model includes a pressure boundary condition which
+is shared between the hot water and chilled water circuits, the two circuits
+being hydronically connected.
```

Comment:
No reason, I had to chose. The two sides being hydronically connected, we cannot have two pressure boundary conditions otherwise the system of equations would be overdetermined.